src/Model/ROI.py

```diff
@@ -26,6 +26,19 @@ def rename_roi(rtss, roi_id, new_name):
     return rtss
 
 
+def delete_list_of_rois(rtss, rois_to_delete):
+    """
+    Call the delete_roi function for each ROI in the given list.
+    :param rtss: Dataset of RTSS.
+    :param rois_to_delete: List of ROI names.
+    :return: Updated RTSS with deleted ROIs.
+    """
+    for item in rois_to_delete:
```

Comment:
item has a particular connotation in DICOM, i.e. a Sequence Item.
So that makes this a bit confusing.


```suggestion
    for roi_name in rois_to_delete:
```